src/CAPNet.Tests/XmlCreatorTests.cs

```diff
@@ -127,14 +127,14 @@ private static Alert CreateMultipleAlertXmlAlert()
                 Source = "source",
                 //   <restriction>restriction</restriction>
                 Restriction = "restriction",
-                //   <code>code</code>
-                Code = "code",
                 //   <note>note</note>
                 Note = "note",
                 //   <references>references</references>
                 
             };
 
+            //<code>code</code>
+            orangeAlert.Codes.Add("code");
```

Comment:
Can you please add one more code, so we have the multiple codes case covered?

Reply:
Hi, I've added one more code.